Sources/CoalView/Components/List/CoalCategoryView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CoalCategoryView.swift
+//
+//
+//  Created by M. Rizki Maulana on 26/09/24.
+//
+
+import SwiftUI
+import CoalModel
+import CoalCore
+
+public struct CoalCategoryView: View {
+  let category: CategoryModel
+  
+  public init(category: CategoryModel) {
+    self.category = category
+  }
+  
+  public var body: some View {
+    VStack(spacing: 8) {
+      ZStack {
+        Color(hex: category.background)
+          .frame(width: 60, height: 60)
+          .cornerRadius(15)
+        
+        icon
+      }
+      
+      Text(category.title)
+        .font(.caption)
+        .foregroundColor(.black)
+    }
+  }
+  
+  private var icon: some View {
+    Group {
+      if category.icon.isValidURL {
+        AsyncImage(url: URL(string: category.icon)) { image in
+          image
+            .resizable()
+            .scaledToFit()
+            .frame(width: 30, height: 30)
+        } placeholder: {
+          RoundedRectangle(cornerRadius: 15)
+            .foregroundColor(.gray)
+        }
+      } else if UIImage(named: category.icon) != nil {
```

Comment:
bisa di refactor jadi gini ?
` else if let iconName = UIImage(named: category.icon) {`

Reply:
Ini gabisa mas, bakalan error gini di **Group** `Type of expression is ambiguous without a type annotation`

Reply:
ok case closed